03LessonThree/in-class-project/daily-planet.html

```diff
@@ -5,6 +5,37 @@
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
+
+      <header>
+
+        <h1>Daily Planet</h1>
+
+        <nav>
+

```

Comment:
great spacing and organization! start getting into the habit of commenting your code :) great job!